release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2643,3 +2643,17 @@ link:https://access.redhat.com/solutions/6324771[{product-title} 4.8.11 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+// TODO: Add this to the end of the appropriate 4.8 z-stream RNs that this check is added with.
+// TODO: Do we need to include this notice in all future z-stream RN as well?
```

Comment:
we should include this note in any z streams where we expect to have admin acks.  E.g. we don't currently know of anything folks would need to ack in 4.9 before heading off to 4.10.  I dunno how likely we are to stumble across anything in that space as 4.10 takes shape, though.  And I dunno if it's worth calling out the possibility of future gates in a 4.9 GA release note?

Reply:
@wking My question here was specific to 4.8 z-streams (not anything as far ahead as 4.9->4.10). So say this goes out in 4.8.13, do i only include this blurb in 4.8.13? Or should it also be included in 4.8.14 and 4.8.15 and so on? 

I wasn't sure if there was a possibility that someone might kind of skip or miss 4.8.13 where this notice was needed going forward to make sure they didn't miss it.

Reply:
if they miss it, their cluster will be alerting `ClusterNotUpgradeable` and if they ask the cluster to update from 4.8 to 4.9 it will block on `Upgradeable=False`.  And folks looking to get to 4.9 are probably not all that focused on 4.8.z release notes anyway.  So I'm personally fine relying on that in-cluster stuff, and not repeating this section in each subsequent 4.8.z's notes.  But I'm also not strongly against repeating this or cross-linking or whatever.